src/provider.js

```
@@ -32,7 +33,9 @@ const validateRevokeBody = (req) => {
 }
 
 const handleError = (res, error, errorMessage) => {
-  console.log(errorMessage, error);
+  if (config.debugMode) {
+    console.log(errorMessage, error);
```

@rodrigopavezi when I see errorMessage I assume it's something unexpected and not the regular info/debug message. Don't know if we really want to hide all unexpected error messages.

---

@rudibravo actually, I think it is a good point. I went through what is have being logged and there is no debugging ones. So I remove the check for the debug mode but left the switch. If we add any more functionality that requires debugging logs we can just use it. Thoughts? @tejohnso ??

---

Yeah  I agree it's best not to hide errors in regular operation. 

If we find later on that for some reason there's a type of error that shows up too often and isn't useful then we can handle silencing that specific case.